File: Backend/LogisticsPlatform.Application/DTOs/Orders/CreateOrderItemDto.cs
public class CreateOrderItemDto
{
    public string Name { get; set; } = string.Empty;
    public string? CustomerReference { get; set; }

    public decimal Quantity { get; set; }
    public string QuantityUnit { get; set; } = "Pallets";
    public bool IsHazmat { get; set; }
    public string? FreightClass { get; set; }
    public string? HazardClass { get; set; }
    public string? IdentificationNumber { get; set; }

    public string? Notes { get; set; }
}

File: Frontend/src/app/core/models/orders/order-items/create-order-item.dto.ts
export interface CreateOrderItemDto {
    name: string;
    customerReference?: string;

    quantity: number;
    actualQuantity?: number;
    status?: string;
    quantityUnit: string;
    handlingQuantity?: number;
    handlingUnit?: string;
    unitNetWeight?: number;
    unitGrossWeight?: number;
    weightUnit?: string;
    length?: number;
    width?: number;
    height?: number;
    dimensionUnit?: string;
    volume?: number;
    volumeUnit?: string;
    minTemperature?: number;
    maxTemperature?: number;
    temperatureUnit?: string;

    isHazmat: boolean;

    freightClass?: string;
    hazardClass?: string;
    identificationNumber?: string;
    declaredValue?: number;
    currency?: string;
    stackable?: boolean;
    copyToLoad?: boolean;

    notes?: string;
}
